utils/utils-lib/src/tracing_telemetry.rs

```diff
@@ -76,98 +50,25 @@ pub enum FmtLayer {
-    let (stdout, stderr) = match fmt_layer {
-        FmtLayer::Stdout => (
-            Some(
-                tracing_subscriber::fmt::layer()
-                    .with_writer(std::io::stdout.with_filter(|meta| meta.target() != EVENT_BUS))
-                    .pretty(),
-            ),
-            None,
-        ),
-        FmtLayer::Stderr => (
-            None,
-            Some(
-                tracing_subscriber::fmt::layer()
-                    .with_writer(std::io::stderr.with_filter(|meta| meta.target() != EVENT_BUS))
-                    .pretty(),
-            ),
-        ),
-        FmtLayer::None => (None, None),
-    };
-
-    // Get the optional eventing layer.
-    let events_layer = match events_url {
-        Some(url) => {
-            let target = filter::Targets::new().with_target(EVENT_BUS, Level::INFO);
-            Some(EventHandle::init(url.to_string(), service_name).with_filter(target))
-        }
-        None => None,
-    };
-
-    let subscriber = Registry::default()
-        .with(tracing_filter::rust_log_filter())
-        .with(stdout)
-        .with(stderr)
-        .with(events_layer);
-
-    match jaeger {
-        Some(jaeger) => {
-            tracing_tags.append(&mut default_tracing_tags(
-                super::raw_version_str(),
-                env!("CARGO_PKG_VERSION"),
-            ));
-            let tracing_tags =
-                tracing_tags
-                    .into_iter()
-                    .fold(Vec::<KeyValue>::new(), |mut acc, kv| {
-                        if !acc.iter().any(|acc| acc.key == kv.key) {
-                            acc.push(kv);
-                        }
-                        acc
-                    });
-            set_jaeger_env();
-
-            global::set_text_map_propagator(TraceContextPropagator::new());
-            let tracer = opentelemetry_jaeger::new_agent_pipeline()
-                .with_endpoint(jaeger)
-                .with_service_name(service_name)
-                .with_trace_config(
-                    opentelemetry::sdk::trace::Config::default()
-                        .with_resource(Resource::new(tracing_tags)),
-                )
-                .install_batch(opentelemetry::runtime::TokioCurrentThread)
-                .expect("Should be able to initialise the exporter");
-            let telemetry = tracing_opentelemetry::layer().with_tracer(tracer);
-            subscriber.with(telemetry).init();
-        }
-        None => subscriber.init(),
-    };
-}
-
 /// Tracing telemetry style.
+#[derive(Debug, Clone, strum_macros::EnumString, strum_macros::AsRefStr)]
+#[strum(serialize_all = "lowercase")]
 pub enum FmtStyle {
     /// Compact style.
     Compact,
     /// Pretty Style
     Pretty,
+    /// JSON Style.
+    Json,
 }
 
+const EVENT_BUS: &str = "mbus-events-target";
+
 /// Tracing telemetry builder.
 pub struct TracingTelemetry {
     writer: FmtLayer,
     style: FmtStyle,
-    colours: bool,
+    colors: bool,
```

Comment:
why modify this naming? :)

Reply:
colors is what the lib also takes. color/colour is US/UK english representation right? So thought of keeping it inline with library

Reply:
Since colour is already in, let's keep as is